packages/amplify-graphql-model-transformer/src/resolvers/rds/resolver.ts

```diff
@@ -160,14 +160,43 @@ export const createRdsLambda = (
   );
 };
 
+/**
+ * Create RDS Patching Lambda function
+ * @param stack Construct
+ * @param apiGraphql GraphQLAPIProvider
+ * @param lambdaRole IRole
+ */
+export const createRdsPatchingLambda = (
+  stack: Stack,
+  apiGraphql: GraphQLAPIProvider,
+  lambdaRole: IRole,
+  environment?: { [key: string]: string },
+  sqlLambdaVpcConfig?: VpcConfig,
+): IFunction => {
+  const { RDSPatchingLambdaLogicalID } = ResourceConstants.RESOURCES;
+  return apiGraphql.host.addLambdaFunction(
+    RDSPatchingLambdaLogicalID,
+    `functions/${RDSPatchingLambdaLogicalID}.zip`,
+    'index.handler',
+    path.resolve(__dirname, '..', '..', '..', 'lib', 'rds-patching-lambda.zip'),
+    Runtime.NODEJS_18_X,
+    [],
+    lambdaRole,
+    environment,
+    Duration.seconds(60),
```

Comment:
How long is this taking in your testing now?

Reply:
About 5-10 seconds. I'm going to add a retry mechanism in a follow-up PR. In that case, we will make first retry after 5 seconds and second retry after 25 seconds. Considering that setting the timeout to 60s.